guides/source/getting_started.md

```diff
@@ -558,7 +558,7 @@ file, and replace its contents with:
 </ul>
 ```
 
-The above code is a mixture of HTML and *ERB*. ERB is a templating system that
+The above code is a mixture of HTML and *ERB*. ERB, short for [Embedded Ruby](https://en.wikipedia.org/wiki/ERuby#:~:text=Embedded%20Ruby%20), is a templating system that
```

Comment:
```suggestion
The above code is a mixture of HTML and *ERB*. ERB, short for [Embedded Ruby](https://en.wikipedia.org/wiki/ERuby), is a templating system that
```

Reply:
Great suggestion! I'll go ahead and make the change.